### PR TITLE
Update announcement.yaml

### PR DESCRIPTION
**Pull request by billy522**
https://17media.slack.com/archives/C0PAD7W5R/p1649657170850599

### CODE REVIEW

File: sta/event-server/announcement.yaml [hunk 1]
```diff
@@ -1,32 +1,82 @@
...
+        \n
+        Rest assured, live-streaming and gifting services will remain available; babycoins and royalty points earned during this maintenance window will be correctly reflected after the downtime.\n
+        \n
+        We apologize for any inconvenience this might cause and thank you for your support.
         pic: "17baobao.jpg"
     TW:
-        title: "活動系統維修公告"
-        description: "親愛的17用戶\n
-          17LIVE APP 預計於 2022/01/19 04:00 (GMT+8) 進行系統維修，並於 2022/01/19 05:00 (GMT+8) 重新開放，期間無法登入與使用，造成您的不便還請見諒，感謝您的支持與諒解，謝謝！ 17LIVE營運團隊敬上。"
+        title: 系統維護公告
```
Comment (brentchang) [inline sta/event-server/announcement.yaml:25]: 好像缺雙引號把它包起來

File: sta/event-server/announcement.yaml [hunk 2]
```diff
@@ -1,32 +1,82 @@
...
+        description: 親愛的用戶您好，感謝您對17Live的支持及愛戴。\n
```
Comment (brentchang) [inline sta/event-server/announcement.yaml:26]: 缺雙引號?

File: sta/event-server/announcement.yaml [hunk 3]
```diff
@@ -1,32 +1,82 @@
...
+        \n
+        ご迷惑をおかけしますが、よろしくお願い致します。\n
+        \n
+        今後とも、17LIVEをよろしくお願いいたします。"
         pic: "17baobao.jpg"
     CN:
-        title: "活動系统维修公告"
-        description: "亲爱的17用户\n
-          17LIVE APP 预计于 2022/01/19 04:00 (GMT+8) 进行系统维修，并于 2022/01/19 05:00 (GMT+8) 重新开放，期间无法登入与使用，造成您的不便还请见谅，感谢您的支持与谅解，谢谢！ 17LIVE营运团队敬上。"
+        title: 系统维护公告
```
Comment (brentchang) [inline sta/event-server/announcement.yaml:67]: 缺雙引號?

File: sta/event-server/announcement.yaml [hunk 4]
```diff
@@ -1,32 +1,82 @@
...
+        description: 亲爱的用户您好，感谢您对17Live的支持及爱戴。\n
```
Comment (brentchang) [inline sta/event-server/announcement.yaml:68]: 缺雙引號?

File: sta/event-server/announcement.yaml (hunk 1 above)
Comment (siansiansu) [reply sta/event-server/announcement.yaml:25]: 雙引號是 force string，這邊應該還是判定成 string，所以應該有沒有加都沒差
......不過還是加一下好了 (怕.jpg)

**Review by siansiansu — APPROVED** [review]

File: sta/event-server/announcement.yaml (hunk 4 above)
Comment (siansiansu) [reply sta/event-server/announcement.yaml]: resolved!

File: sta/event-server/announcement.yaml (hunk 3 above)
Comment (siansiansu) [reply sta/event-server/announcement.yaml]: resolved!

File: sta/event-server/announcement.yaml (hunk 2 above)
Comment (siansiansu) [reply sta/event-server/announcement.yaml]: resolved!

File: sta/event-server/announcement.yaml (hunk 1 above)
Comment (siansiansu) [reply sta/event-server/announcement.yaml]: resolved!

**Pull request merged**